config/data_funcs.go

```diff
@@ -156,13 +157,17 @@ func unmarshalOrReturnString(value string) (result interface{}) {
 
 func parseCmdline(cmdLine string) map[interface{}]interface{} {
 	result := make(map[interface{}]interface{})
+	params, err := shellwords.Parse(cmdLine)
```

Comment:
Did you mean?

```
p := shellwords.NewParser()
p.ParseEnv = false
p.ParseBacktick = false
params, err := p.Parse(cmdLine)
```
Since you duplicated same line twice
```
p.ParseEnv = false
p.ParseEnv = false
```

Yes, it makes sence, corrected